pwa/src/components/logs/logTable/logTable.tsx

```diff
@@ -183,7 +192,7 @@ export const LogTable: React.FC<LogTableProps> = ({ entityId }) => {
                       </tr>
                       <tr>
                         <th>Response time</th>
-                        <td>{log?.responseTime.toString() + ' seconds'}</td>
+                        <td>{millisecondsToSeconds(log?.responseTime).toString() + ' seconds'}</td>
```

Comment:
I think this one should get the same structure as the other one (so 2342ms (2s))